Library/Homebrew/extend/ENV/std.rb

```diff
@@ -287,7 +287,7 @@ def cxx11
     if compiler == :clang
       append "CXX", "-std=c++11"
       append "CXX", "-stdlib=libc++"
-    elsif compiler =~ /gcc-(4\.(8|9)|5)/
+    elsif compiler =~ /gcc-(4\.(8|9)|5|6)/
```

Comment:
Can you replace `5|6` with `[5-6]` to make it easier to extend this in the future?